### DISCUSSION TITLE
Add new parameter for differenceEuclidean to use Euclidean distance for hue-based color spaces

### DISCUSSION THREAD

**Discussion by lulunac27a** [Evercoder/culori#244] [Ideas]
I want to add the parameter to use the Euclidean distance for the [differenceEuclidean](https://culorijs.org/api/#differenceEuclidean) function instead of hue based chord lengths for hue-based cylindrical color spaces like HSL, LCHab with the hue difference clamped from -180 to 180 degrees, not from -360 to 360 degrees.

**Comment by danburzo** [comment]
I wouldn’t overload the Euclidean distance function any further, but it should be easy enough to fork and simplify it into a separate userland function.
